app/graphql/caching-utils.ts

```diff
@@ -0,0 +1,22 @@
+/** As Lindas supports caching requests per given id, we can utilize this function
+ * to query endpoint that will cache the results per given cube iri.
+ *
+ * This means that if we query the same endpoint with the same cube iri, we will
+ * get the cached results, unless the cache is invalidated.
+ *
+ * If cubeIri is not provided, we default to a general cached endpoint, which
+ * has much higher cache invalidation time (two minutes as of writing this comment).
+ */
+export const getCachedSparqlUrl = ({
+  endpointUrl,
+  cubeIri,
+}: {
+  endpointUrl: string;
+  cubeIri: string | undefined;
+}) => {
+  if (cubeIri) {
+    return `${endpointUrl}/${cubeIri}`;
+  }
+
+  return endpointUrl;
+};
```

Comment:
I wonder if we should have a environment variable that controls this behavior.

1. If I understood correctly, this will only work on Prod and not on test / Int
2. We could easily activate this code once the behavior lands, instead of waiting to merge, it would only be a flip of environment variable.

LINDAS_SUPPORTS_CUBE_QUERIES or something like this ? What do you think ?